Makefile

```diff
@@ -46,3 +46,7 @@ test:
 .PHONY: build
 build:
 	docker build -t digital_roadmap:latest -f Containerfile .
+
+.PHONY: tests
+tests:
+	$(VENV_DIR)/bin/pytest
```

Comment:
I think this does the same thing as the test target.

Reply:
That's right, I missed the target `test`. As I was used to using `tests` (which missed), I didn't look for other variants

Removed :)